### PR TITLE
Change excluding terms into inclusive alternatives

### PR DESCRIPTION
**Pull request by kazydek**
**Description**

Changes proposed in this pull request:

- Changed excluding terms into inclusive alternatives

**Related issue(s)**
See also #https://github.com/kyma-project/community/issues/515

### CODE REVIEW

File: docs/rafter/06-05-clusterassetgroup.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ spec:
       mode: package
       parameters:
         disableRelativeLinks: "true"
-      url: https://github.com/kyma-project/kyma/archive/master.zip
+      url: https://github.com/kyma-project/kyma/archive/main.zip
```
[the remark this answers is not in the document]
Comment (Halamix2) [reply docs/rafter/06-05-clusterassetgroup.md:37]: https://github.com/kyma-project/kyma/archive/master.zip is not working anymore

[the remark this answers is not in the document]
Comment (kazydek) [reply docs/rafter/06-05-clusterassetgroup.md:37]: Yes, `master.zip` returns `404: Not Found`

**Review by klaudiagrz — APPROVED** [review]

**Review by a-thaler — APPROVED** [review]

**Pull request merged**